core/server/services/url/utils.js

```diff
@@ -324,7 +327,8 @@ function urlFor(context, data, absolute) {
         }
 
         if (data && data.version) {
-            apiPath = getApiPath(data.version, data.admin);
+            let versionType = data.admin ? 'admin' : 'content';
```

Comment:
🤔

Reply:
the `context` in `urlFor` method is still passing `admin:true` logic, which is the one used in all tests right now. Since `getApiPath` method changed, I updated this to handle it correctly, as changes in `urlFor` can be a separate PR but is not related :)

Reply:
@rishabhgrg

Reply:
Oops missed this before. So `data.admin` is passed from some tests, for ex. [here](https://github.com/TryGhost/Ghost/blob/master/core/test/unit/services/url/utils_spec.js#L539) to `urlFor` context. Did I get the question right?

Reply:
But these tests should be updated to use `type`?

Reply:
Ah ok I see the confusion. So in this PR, I have updated `getApiPath` method to use `version` and `type` logic, but haven't touched `urlFor` method which still uses admin as is, and `urlFor` uses `getApiPath`. Tests only use `urlFor`, so how they pass admin hasn't changed yet, I was wondering that should be separate PR which changes only `urlFor` method and not part of this. Can include those changes here itself though if you are saying that makes more sense :)

Reply:
💃